cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app_complex/templates/java/${variables.rootPackage}/${variables.component}/logic/api/usecase/UcManage${variables.entityName}.java.ftl

```diff
@@ -2,6 +2,17 @@ package ${variables.rootPackage}.${variables.component}.logic.api.usecase;
 
 import ${variables.rootPackage}.${variables.component}.logic.api.to.${variables.entityName}Eto;
 
+<#assign compositeIdVar = false>
+<#list pojo.fields as field>
+	<#if field.type?starts_with("Composite")>
+		<#assign compositeIdVar = true>
+		<#assign compositeIdTypeVar = field.type>
+	</#if>
+</#list>
```

Comment:
missing refactoring

Reply:
Changes done and committed. Re-Review required.